.github/workflows/docker-build.yml

```diff
@@ -31,11 +41,36 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: tag version
+        uses: nowsprinting/check-version-format-action@v3
+        id: tag_version
+        with:
+          prefix: 'v'
+
+      - name: tag version extract
+        run: |
+          process.env.VERSION = ${{ steps.version.outputs.full }}
+        if: steps.tag_version.outputs.is_valid == 'true'
```

Comment:
If the `nowsprinting/check-version-format-action@v3` was able to get the tag version and is valid, then we will use that one. If not, we will get it from the branch name.
@evan-forbes what do you think? should we test this?